test/test_token.js

```diff
@@ -73,7 +73,12 @@ describe('test create token', function () {
     let bytes = tra.toBytes();
     let signature = acc.getSignature(bytes);
     let send_tx = tra.toJsonForSendingTx(signature);
-
+    let create_parse_function_data = convert.parseFunctionData(send_tx['initData']);
+    let create_original_data = BigNumber(contractTx['initData']['amount']).multipliedBy(contractTx['initData']['unity']);
+    it('unit test for parseFunctionData when destroy token', function() {
+        expect(create_original_data.isEqualTo(create_parse_function_data[0])).to.be.equal(true);
```

Comment:
Simplify to `expect(A).to.be.equal(B)` instead of `expect(A==B).to.be.equal(true)`

Reply:
If type is different, convert to string and then compared.

Reply:
I have tried in this way before, but it didn't work. 
![image](https://user-images.githubusercontent.com/32029023/66805616-044fa600-ef58-11e9-835e-2cf2489fdcaa.png)

Reply:
![image](https://user-images.githubusercontent.com/32029023/66805672-277a5580-ef58-11e9-9f10-0b358a0e61cd.png)

Reply:
I can change them to string and use expect(A).to.be.equal(B)